### ISSUE TITLE
Matrix editing shortcuts are not working

### ISSUE THREAD

**Issue by Kydyralin** [arnog/mathlive#736]
## Description

Since **version 0.59.0** Matrix editing shortcuts are not working.

## Steps to Reproduce

1. run `npm start`
2. open [http://localhost:8080/examples/basic/index.esm.html](http://localhost:8080/examples/basic/index.esm.html) page
3. Type `()` and inside parentheses try to enter matrix editing shortcuts

    | Shortcut | Command |
    |:---|:---|
    | **ctrl**/**cmd** + **Return**/**Enter** | `addRowAfter` |
    | **ctrl**/**cmd** + **shift** + **Return**/**Enter** | `addRowBefore` |
    | **ctrl**/**cmd** + **;** | `addRowAfter` |
    | **ctrl**/**cmd** + **shift** + **;** | `addRowBefore` |
    | **ctrl**/**cmd** + **,** | `addColumnAfter` |
    | **ctrl**/**cmd** + **shift** + **,** | `addColumnBefore` |

### Actual Behavior

Nothing happens, the editor doesn't react to commands.

No error in the console was thrown.

### Expected Behavior

Commands were applied correctly to the matrix (e.g. adding row below on pressing **ctrl**/**cmd** + **Enter** etc. ) 

[Is this a regression: did it use to work in a previous version?]
Yes, it worked correctly in **version 0.58.0** 

### Environment

**Operating System**
[macOS, Windows, iOS. Include the version if possible]
macOS, Windows 10

**Browser** [Safari, Chrome, IE, Firefox, etc...]
Safari, Chrome, Edge

**Comment by Kydyralin** [comment]
Hi!

I'd like to help you out with the fix of this issue but I don't know the starting point to investigate. If you could point at the place in the codebase where I should take a look it would be really helpful.

**Comment by Kydyralin** [comment]
After some investigation I found out that commit `44f0c5d16bbc0096d48804c44b0a36291954107f` has changes on **array.ts** file where `convertParentToArray` and 
`addCell` functions' implementation are commented out.

Is there any reason to doing so or it was just forgotten to uncomment?

**Comment by arnog** [comment]
These functions are commented out because they need to be rewritten to work with the new selection model. Getting selection in matrix to work (again) is leftover work from the implementation of the new selection model. It's on my todo list.

**Comment by alexprey** [comment]
Hi @arnog do you have any progress there?

**Comment by arnog** [comment]
not yet, still on my todo list

**Comment by 96imranahmed** [comment]
Hey @arnog - is this still on the to-dos?

**Comment by arnog** [comment]
Yes, still on the to-do list...

**Comment by amarcolini** [comment]
Testing this on the latest version of mathlive, and the only shortcuts that work are `ctrl + enter`, `ctrl + ;`, `ctrl + shift + ;`.

**Comment by arnog** [comment]
Could you be more specific and indicate which shortcuts you expected to work that didn’t?

**Comment by amarcolini** [comment]
All the shortcuts in the original issue should work, unless I am mistaken. Notably, the only shortcuts that work are for adding rows, not columns. I tried this on the example on the mathlive website, my own example, and by building from source.

**Comment by arnog** [comment]
:) This issue was filed three years ago. Things have changed. BTW, in general you're better off filing a new issue rather than commenting on a closed one. I happened to notice your comment, but that's by luck.

The matrix editing keyboard shortcuts are `option+tab`, `option+return`, `option+shift+tab` and `option+shift+return`. In theory they should be documented [here](https://cortexjs.io/mathlive/reference/keybindings/#matrix-editing), but I just noticed the documentation is not up to date.
